sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Reshuffle.java

```diff
@@ -65,6 +66,11 @@ private Reshuffle() {}
     return new ViaRandomKey<>();
   }
 
+  @Experimental
+  public static <T> Reparallelize<T> reparallelize() {
```

Comment:
It is unclear to a user why they would choose this implementation over viaRandomKeys

Reply:
Yes I will improve the javadoc, maybe move some of the details in the impl. comment below.